tests/bug-5220/bug-5220-6/test.yaml

```diff
@@ -0,0 +1,15 @@
+requires:
+  pcap: false
+  min-version: 7
+
+args:
+  - --engine-analysis
+
+exit-code: 0
+
+checks:
+  - filter:
+      filename: patterns.json
+      count: 1
+      match:
+        buffers[0].patterns[0].pattern: "POST"
```

Comment:
@catenacyber w.r.t. your comment https://github.com/OISF/suricata-verify/pull/1719#discussion_r1533941230
Does this look ok to you?
I only see the info that POST is the final fast pattern in `rules_analysis.txt`..

Reply:
ok for me